src/gamma.jl

```diff
@@ -599,17 +599,15 @@ See also [`loggamma`](@ref).
 logabsgamma(x::Real) = _logabsgamma(float(x))
 
 function _logabsgamma(x::Float64)
-    signp = Ref{Int32}()
-    y = ccall((:lgamma_r,libopenlibm),  Float64, (Float64, Ptr{Int32}), x, signp)
-    return y, Int(signp[])
+    y, s = _lgamma_r(x)
```

Comment:
delete these and just rename the main implimentation to `_logabsgamma`?

Reply:
Indeed, streamlines interface.